core/schemata/json-schema/artifacts/entityRepresentationImage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "entityRepresentationImage",
+  "type": "object",
+  "title": "Entity Image",
+  "properties": {
+    "imageId": {
```

Comment:
@setaman maybe we should name this something like `entityRepresentationImageId`, because it is a root level property. Same for Line 17.